transport-native-kqueue/src/main/java/io/netty/channel/kqueue/KQueueEventLoop.java

```diff
@@ -76,6 +77,8 @@ public Integer call() throws Exception {
     private volatile int wakenUp;
     private volatile int ioRatio = 50;
 
+    static final long MAX_SCHEDULED_DAYS = 356 * 3;
```

Comment:
please note that I used this as it seems to work on linux / Windows and macOS.

Reply:
Why do we care about windows and linux here? This is in the kqueue transport.

Reply:
https://www.freebsd.org/cgi/man.cgi?query=kqueue&sektion=2#end

>  The timeout value is limited to 24	hours; longer timeouts will be
     silently reinterpreted as 24 hours.

Do we need to do anything for kqueue? IIRC we verified that scheduled tasks are actually expired even if we are woken up as a result of timeout.

Reply:
@Scottmitch sorry this was confusing. I tested the value with NIO on linux, windows and macOs ;)

Reply:
@Scottmitch oh my ... This is a nice "bug". I wonder if this is still true and why they not just fixed it... any idea ?

Reply:
s/356/365/ ?